acra-censor/acra-censor_test.go

```diff
@@ -339,7 +339,7 @@ func testWhitelistColumnsPattern(t *testing.T) {
 		// different table
 		"SELECT * FROM testTable1 ORDER BY 1",
 		// different columns in select expressions
-		"SELECT A FROM testTable ORDER BY (case when f1 then 1 when f1 is null then 2 else 3 end)",
+		//"SELECT A FROM testTable ORDER BY (case when f1 then 1 when f1 is null then 2 else 3 end)",
```

Comment:
Because this query shouldn't be blocked by whitelist (and same is true for other queries with defined columns when pattern contains '*' in SelectExprs). We have:

pattern - "SELECT * FROM testTable ORDER BY %%COLUMN%%"
query - "SELECT A FROM testTable ORDER BY (case when f1 then 1 when f1 is null then 2 else 3 end)"

and * means that we accept any column in SelectExprs. So if query matches pattern, whitelist should pass it. Please, look at this: https://github.com/cossacklabs/acra/blob/master/acra-censor/acra-censor_test.go#L548 - pattern with * matches query with a, b columns and whitelist passes it